examples/apollo-service-example/src/main/java/com/spotify/zoltar/examples/apollo/IrisPrediction.java

```diff
@@ -71,27 +57,17 @@
   public static void configure(final URI modelDirUri, final URI settingsUri) throws IOException {
     final FeatureSpec<Iris> irisFeatureSpec = IrisFeaturesSpec.irisFeaturesSpec();
     final String settings = new String(Files.readAllBytes(Paths.get(settingsUri)));
-    final ModelLoader<TensorFlowModel> modelLoader =
-        Models.tensorFlow(modelDirUri.toString());
 
-    final ExtractFn<Iris, Example> extractFn =
-        FeatranExtractFns.example(irisFeatureSpec, settings);
-
-    final TensorFlowPredictFn<Iris, Long> predictFn = (model, vectors) -> {
-      final List<CompletableFuture<Prediction<Iris, Long>>> predictions =
-          vectors.stream()
-              .map(vector -> CompletableFuture
-                  .supplyAsync(() -> predictFn(model, vector.value()))
-                  .thenApply(v -> Prediction.create(vector.input(), v)))
-              .collect(Collectors.toList());
-      return CompletableFutures.allAsList(predictions);
-    };
-
-    predictor = Predictor.create(modelLoader, extractFn, predictFn);
+    predictor = new SimpleTensorFlowPredictorBuilder<Iris>()
```

Comment:
Maybe `SimpleTensorFlowPredictorBuilder` should be exposed as part of `zoltar-models` in `Predictors` class, kinda like:

```
Predictors.simpleTensorFlow()
```

This way we can add more predictors to `Predictors` as needed, and make them easier to discover. In that case we might also change `zoltar-models` into something like `zoltar-api` or something like that. WDYT?